lib/datasource/pypi/index.spec.ts

```diff
@@ -92,18 +96,28 @@ describe('datasource/pypi', () => {
         .scope('https://second-index/foo')
         .get('/azure-cli-monitor/json')
         .reply(200, JSON.parse(res1));
+      httpMock
+        .scope('https://third-index/foo')
+        .get('/azure-cli-monitor/json')
+        .reply(200, JSON.parse(res2));
       const config = {
         registryUrls: [
           'https://custom.pypi.net/foo',
           'https://second-index/foo',
           'https://third-index/foo',
         ],
       };
-      await getPkgReleases({
+      const res = await getPkgReleases({
         ...config,
         datasource,
         depName: 'azure-cli-monitor',
       });
+      expect(
+        res.releases.reduce((x, y) => (semver.gt(x.version, y.version) ? x : y))
+      ).toMatchObject({
+        version: '0.2.15',
+        releaseTimestamp: '2019-06-18T13:58:55',
+      });
```

Comment:
You don't need to reduce, the `res.releases` should be ordered ascending, so `res.releases.pop()` should be latest version.

Reply:
Alright, I wasn't sure if it was guaranteed to be ordered and I didn't had much time to dig deep into it 🙂 
Thanks!

Reply:
Changed 🙂